### PR TITLE
Add README and common Daytona docs

### LINKED ISSUE #3: Remake the README in accordance with org-wide layout
For consistency, the README should take style cues from the main Daytona project repo.

### PR DESCRIPTION
**Pull request by osslate**
In addition to README, add LICENSE + corresponding field in `package.json`, the CODE_OF_CONDUCT, and the SECURITY information.

Closes #3 and #12.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,92 @@
...
+    Background and conceptual information about Daytona and DEMs.
+*   __Installation__  
+    Information around installing Daytona under different environments.
+*   __Reference__  
+    Technical and architectural information about Daytona.
+*   __Usage__  
+    Practical information on using Daytona.
+
+## Contributing
+Daytona is licensed under the [Elastic License 2.0](LICENSE). If you would like to contribute to the software, you must:
```
Comment (Tpuljak) [inline README.md:75]: The License is Apache.

File: README.md [hunk 2]
```diff
@@ -1,27 +1,92 @@
...
+
+1. Read the Developer Certificate of Origin Version 1.1 (https://developercertificate.org/)
+2. Sign all commits to the Daytona project.
+
+This ensures that users, distributors, and other contributors can rely on all the software related to Daytona being contributed under the terms of the [License](LICENSE). No contributions will be accepted without following this process.
+
+## License
+This repository contains the Daytona Enterprise Demo installer, covered under the [Apache License 2.0](LICENSE), except where noted (any Daytona logos or trademarks are not covered under the Apache License 2.0, and should be explicitly noted by a LICENSE file.)
```
Comment (Tpuljak) [inline README.md:83]: C/p from our other repo. Update to sync with Daytona Docs.

**Review by Tpuljak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (osslate) [reply README.md]: Fixed in [4560ea5](https://github.com/daytonaio/docs/pull/23/commits/4560ea576d9e99e8cdce0b6dec6054ca01e6bfaf)

File: README.md (hunk 2 above)
Comment (osslate) [reply README.md]: Updated in [4560ea5](https://github.com/daytonaio/docs/pull/23/commits/4560ea576d9e99e8cdce0b6dec6054ca01e6bfaf).

**Review by Tpuljak — APPROVED** [review]

**Review by ivan-burazin — APPROVED** [review]

**Pull request merged**